make-all.sh

```diff
@@ -95,11 +95,8 @@ fi
 
 echo "[*] Building bddisasm ..."
 cd libs/bddisasm
-if [[ "$is_windows" == "y" ]]; then
-    msbuild.exe bddisasm/bddisasm.vcxproj -p:Configuration=Release -p:Platform=x64
```

Comment:
taking your word that `cmake --build build --target bddisasm` is sufficient to get it built on windows, at least 😁